src/azul/openapi/schema.py

```diff
@@ -192,12 +193,32 @@ def enum(*items: PrimitiveJSON, type_: TYPE = None) -> JSON:
             "bar"
         ]
     }
-    """
 
+    >>> assert_json(enum(items=['a']))
+    {
+        "type": "string",
+        "enum": [
+            "a"
+        ]
+    }
+    >>> assert_json(enum(*['a'], items=['b']))
+    Traceback (most recent call last):
+    ...
+    azul.RequirementError
+    """
+    if items is None:
+        # jsonschema validator package rejects tuples, so convert to list.
+        items_ = list(items_)
+    else:
+        require(len(items_) == 0)
+        # TODO: This isinstance check needs to be looser. allow mappings.
+        #    https://github.com/DataBiosphere/azul/issues/2375
+        # require(isinstance(items, list))
+        items_ = items
```

Comment:
Can we invert this assigment?

Reply:
Sure, is this to make the code easier to read? I changed the positional parameter to `items` and the keyword arg to`items_`, to allow the conditional/inspections to remain the same.

Reply:
Keyword argument should remain `items`.

Reply:
Ok let me take a stab at it, I'll add the changes as a separate fixup.